src/autoscaler/Makefile

```
@@ -25,20 +25,34 @@ export GO111MODULE=on
 GINKGO_OPTS = -r --race --require-suite --randomize-all --cover ${OPTS}
 GINKGO_VERSION = v$(shell cat ../../.tool-versions | grep ginkgo  | cut --delimiter=' ' --fields='2')
 
-# The presence of the subsequent directory indicates wheather the fakes still need to be generated
+
+# ogen generated OpenAPI clients and servers
+openapi-generated-clients-and-servers-dir := ./helpers/apis/scalinghistory
+openapi-spec-path := ../../api
+openapi-specs-list = $(wildcard ${openapi-spec-path}/*.yaml)
+
+openapi-generated-clients-and-servers-files := $(wildcard ${openapi-generated-clients-and-servers-dir}/*.go)
```

```suggestion
openapi-generated-clients-and-servers-files = $(wildcard ${openapi-generated-clients-and-servers-dir}/*.go)
```
Without the `:` to evaluate it each time.